collectd/files/collectd_logfile.conf

```diff
@@ -0,0 +1,10 @@
+<LoadPlugin logfile>
```

Comment:
This is unrelated to this PR but
1/ this doesn't seem to be used anywhere by default so collectd isn't logging anything.
2/ the configuration files should be correctly ordered, eg you want the logging plugin to be loaded first to be able to troubleshoot collectd configuration problems [0].

[0] https://github.com/voxpupuli/puppet-collectd/pull/166#issuecomment-50591413